mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -44,7 +44,8 @@ private[shared] object SharedParamsCodeGen {
         " probabilities. Note: Not all models output well-calibrated probability estimates!" +
         " These probabilities should be treated as confidences, not precise probabilities",
         Some("\"probability\"")),
-      ParamDesc[String]("varianceCol", "Column name for the biased sample variance of prediction"),
+      ParamDesc[String]("varianceCol", "Column name for the biased sample variance of prediction",
+        Some("\"variance\"")),
```

Comment:
I'd like this change, although we did not output variance column by default at #8866. Consider that we have default value for ```probabilityCol```, I think it's ok to add default value for ```varianceCol```.